portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -32,25 +32,30 @@
   /** Delete comment servlet, allow a user to delete their own comments. */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Get the id of which comment to delete.
+    // Get the key of which comment to delete.
```

Comment:
How is this change related to a map?

Reply:
This change is related to my last PR and is part of the branch that it came from so I'm not sure why it is showing as different here.

Reply:
Could you possibly rebase this after getting the other PR merged to master to see if this change disappears from the diff?

Reply:
Yes I will try pull from master and then create another pull request to merge this branch into master.